ci/autoscaler/scripts/register-broker.sh

```diff
@@ -14,19 +13,20 @@ popd
 
 cf api "https://api.${system_domain}" --skip-ssl-validation
 
-CF_ADMIN_PASSWORD=$(credhub get -n /bosh-autoscaler/cf/cf_admin_password -q)
-cf auth admin "${CF_ADMIN_PASSWORD}"
+cf_admin_password=$(credhub get -n /bosh-autoscaler/cf/cf_admin_password -q)
+cf auth admin "${cf_admin_password}"
 
 set +e
-SERVICE_BROKER_EXISTS=$(cf service-brokers | grep -c "${service_broker_name}.${system_domain}")
+existing_service_broker=$(cf service-brokers | grep "${service_broker_name}.${system_domain}" |  cut -d' ' -f1)
```

Comment:
hmmm ... not sure this achieves any more except call cut ?

Reply:
it helps the delete-service-broker command to delete the spefic broker name serched by the broker url